File: UNIFEI - MEUS PROJETOS/PROJETO_CIC133_C_JAVA_LISP_PROLOG/JAVA/mmc/mmc.java
import java.util.Scanner;

public class  mmc{
    // Algoritmo de Euclides iterativo
    private static int mdc(int a, int b) {
        if (a != b) {
            if (b > a) {
                int aux = b;
                b = a;
                a = aux;
            }
            while (b != 0) {
                int r = a % b;
                a = b;
                b = r;
            }
        }
        return a;
    }

    // Algoritmo do MMC
    private static int mmc(int a, int b) {
        return a * (b / mdc(a, b));
    }

    public static void main(String[] args) {
        char sair = 's';
        Scanner dado = new Scanner(System.in);
        System.out.println(" \nO minimo multiplo comum (MMC) de dois numeros.");

        while (sair == 's') {
            System.out.print(" Entre com o primeiro Numero: ");
            int n1 = dado.nextInt();
            System.out.print(" Entre com o segundo Numero: ");
            int n2 = dado.nextInt();
            System.out.println(" O MMC (" + n1 + " ," + n2 + ") = " + mmc(n1, n2));
            System.out.print(" Se deseja continuar tecle s ou para sair qualquer tecla: ");
            sair = dado.next().charAt(0);
        }

        dado.close();
    }
}

File: UNIFEI - MEUS PROJETOS/PROJETO_CIC133_C_JAVA_LISP_PROLOG/C/mmc.c
// o mínimo múltiplo comum (MMC) de dois inteiros a e b é o menor inteiro positivo que é múltiplo simultaneamente de a e de b.
#include <stdio.h>

//maximo divisor comum
int mdc(int a, int b){
    if (a != b) {
        if (b > a) {
            int aux = b;
            b = a;
            a = aux;
        }
        while (b != 0) {
            int r = a % b;
            a = b;
            b = r;
        }
    }
    return a;
}

//minimo divisor comum
int mmc(int a, int b) {
   return  (a * b)/mdc(a, b);
}

int main(void){
    int n1, n2;
    char sair = 's';    
   
    printf("\n O minimo multiplo comum (MMC) de dois numeros.\n"); 
  
   while(sair == 's'){
        printf("\n Entre com o primeiro Numero: ");
        scanf(" %d", &n1);
        printf(" Entre com o segundo Numero: ");
        scanf(" %d", &n2);

        printf(" O MMC (%d , %d ) = %d.\n", n1, n2, mmc(n1, n2));
        printf(" Se deseja continuar tecle s ou para sair qualquer tecla: ");
        scanf(" %c", &sair);
   }
   return 0;
}